### PR TITLE
fix: workaround for chrome issue

### PR DESCRIPTION
**Pull request by slavik-lvovsky**

### CODE REVIEW

File: packages/backend/src/yeomanui.ts [hunk 1]
```diff
@@ -584,6 +586,24 @@ export class YeomanUI {
     });
   }
 
+  // TODO: Remove this workaround after the rootcause and fix are found for:
+  // https://sapjira.wdf.sap.corp/browse/DEVXBUGS-8741
```
Comment (alex-gilin) [inline packages/backend/src/yeomanui.ts:590]: this is public repo - please consider to remote an internal references ....

File: packages/backend/src/yeomanui.ts [hunk 2]
```diff
@@ -584,6 +586,24 @@ export class YeomanUI {
...
+  private onGenInitializing(gen: any) {
```
Comment (alex-gilin) [inline packages/backend/src/yeomanui.ts:591]: whether you indeed need this gen argument ? it exists on this, isn't it ?

File: packages/backend/src/yeomanui.ts (hunk 1 above)
Comment (ira-gordin-sap) [inline packages/backend/src/yeomanui.ts:590]: Link to internal jira.

File: packages/backend/src/yeomanui.ts [hunk 3]
```diff
@@ -584,6 +586,24 @@ export class YeomanUI {
...
+    const genMethodName = "initializing";
+    const originalPrototype = Object.getPrototypeOf(gen);
+    const originalGenWriting = _.get(originalPrototype, genMethodName);
+    if (!originalGenWriting) {
```
Comment (alex-gilin) [inline packages/backend/src/yeomanui.ts:595]:
```suggestion
    if (!_.has(originalPrototype, genMethodName)) {
```

**Review by alex-gilin — CHANGES_REQUESTED** [review]

**Review by alex-gilin — COMMENTED** [review]
may be it worth to write some test for this case ?

**Pull request closed without merging**